modules/ROOT/pages/dashboards.adoc

```diff
@@ -24,6 +24,23 @@ Custom dashboards in Anypoint Monitoring can bring together important metrics an
 
 For example, you can compare live data with historic data to detect anomalies and expedite the troubleshooting process.
 
+[[no-data-scenarios]]
+== No Data Scenarios for Dashboards
+
+A dashboard in Anypoint Monitoring might return "No Data" instead of metrics. The following scenarios commonly cause the "No Data" message to appear:
+
+* You selected an incorrect time range.
++
+For example, the app started yesterday, and you selected a day from the past week as your time range.
+* You removed the Monitoring property was removed from runtime.
```

Comment:
I think there is an extra 'removed' in this sentence.

Reply:
```suggestion
* You removed the Monitoring property from the runtime.
```